src/utils/daemon.js

```diff
@@ -9,6 +9,10 @@ export default async function createDaemon (opts) {
   opts.flags = opts.flags || []
   opts.keysize = opts.keysize || 4096
 
+  if (!opts.flags.includes('--migrate=true')) {
```

Comment:
This seems like a suprising pattern. If you pass in an flags, this code will modify it. It'd be less suprising if we add `--migrate=true` in the default flags array. If the caller chooses to modify it then they should pass all the flags they want.

```js
opts.flags = opts.flags || ['--migrate=true']
```

Reply:
Yes, you're right. Will add that and `--routing=dhtclient` that @lidel mentioned. That one got lost during revamp haha